src/preview/webview.ts

```diff
@@ -301,9 +301,7 @@ export class SwingWebView {
 
     const scriptType = this.isJavaScriptModule
       ? "module"
-      : this.manifest && this.manifest.scriptType
-      ? this.manifest.scriptType
-      : "text/javascript";
+      : (this.manifest?.scriptType || "text/javascript");
```

Comment:
This was not _really_ necessary but I found this while I was chasing down the error and the nested ternary was confusing me too much :)